widget/index.html

```diff
@@ -51,5 +51,8 @@
   </div>
 </div>
 <div view-switcher=""></div>
+<script>
+buildfire.appearance.titlebar.show()
```

Comment:
this will show the titlebar even when the plugin is a home plugin, we sure we want that?